Queue/PopulateProcessor.php

```diff
@@ -102,7 +107,7 @@ public static function getSubscribedCommand(): array
         return [
             'command' => Commands::POPULATE,
             'queue' => Commands::POPULATE,
-            'prefix_queue' => false,
+            'prefix_queue' => self::$config['prefix_queue'],
```

Comment:
I better approach is to use the container tag with attributes

Reply:
@makasim Thanks, I've noticed that `self::$config` will be empty in `BuildCommandSubscriberRoutesPass` [line 55](https://github.com/php-enqueue/enqueue/blob/c2a32569a6df81c060d9b497b63b6a96b1968ee1/Symfony/Client/DependencyInjection/BuildCommandSubscriberRoutesPass.php#L55) so I need to find a better solution.

Right now I don't have enough time, so I'll set it to true and will decline or move this PR to the draft